### ISSUE TITLE
page title incorrect after vote change

### ISSUE THREAD

**Issue by JoeCohen** [MushroomObserver/mushroom-observer#1634]
Minor cosmetic bug. 
To replicate:
- Go to any Observation (ex: http://localhost:3000/516551)
- Change vote 
Result HTML `<head><title>` is messed up.
In my example:
```html
<title>&lt;span class="smaller"&gt;Observation &lt;span&gt;516551:&lt;/span&gt;&lt;/span&gt; &lt;a class="obs_consensus_naming_link_25237" href="/names/25237"&gt;&lt;b&gt;&lt;i&gt;Amanita gemmata&lt;/i&gt;&lt;/b&gt; (Fr.) Bertill.&lt;/a&gt;</title>
```
instead of
```html
<title>Mushroom Observer: Observation 516551: Amanita gemmata (Fr.) Bertill.</title>
```
So we see this:
<img width="705" alt="Screenshot 2023-09-25 at 8 54 23 AM" src="https://github.com/MushroomObserver/mushroom-observer/assets/45460/02d96501-6e62-4a79-bce2-2c976573f37b">

**Comment by nimmolo** [comment]
Got it. Thanks for documenting, Joe. It's an easy fix.

**Comment by nimmolo** [comment]
...Or maybe not. @JoeCohen are the autocompleters working in the modal for you?

They're not for me, and iirc they were working when that PR merged.

**Comment by nimmolo** [comment]
Should be fixed in #1635
